components/resources/brave_vpn_strings.grdp

```diff
@@ -99,11 +99,15 @@
   <message name="IDS_BRAVE_VPN_FEATURE_5" desc="Key feature for selling advertising">
     24/7 support
   </message>
-
+  <message name="IDS_BRAVE_VPN_SUPPORT_TICKET_FAILED" desc="Label to show failed response from support ticket creation request">
+    Failed to create support ticket, check your connection.
+  </message>
   <message name="IDS_BRAVE_VPN_LOADING" desc="Label to show below loader icon">
     One moment while loading…
   </message>
-
+  <message name="IDS_BRAVE_VPN_PURCHASE_FAILED " desc="Label to show when purchase status failed">
+    Unable to check the status of your purchase, probably a network problem.
```

Comment:
How about "Unable to check the status of your purchase. This could be caused by a network problem."

Reply:
done